transformers/integration_tests/median_lift_test.go

```diff
@@ -91,7 +91,7 @@ var _ = Describe("MedianLift EventTransformer", func() {
 		header, headerErr := persistHeader(db, blockNumber, blockChain)
 		Expect(headerErr).NotTo(HaveOccurred())
 
-		medianWbtcAddress := constants.GetContractAddress("MEDIAN_WBTC")
+		medianWbtcAddress := constants.GetContractAddress("MEDIAN_WBTC_1_0_6")
```

Comment:
maybe worth using `test_data.MedianWbtcAddress()` so we don't need to make updates here to change the contract name?

Reply:
👍 - should do more of that probably TBH. There's a lot of string duplication.